### PR TITLE
[CDAP-17470] Replication address UX review

### PR DESCRIPTION
**Pull request by elfenheart**
JIRAs:
https://cdap.atlassian.net/browse/CDAP-17432
https://cdap.atlassian.net/browse/CDAP-17328
https://cdap.atlassian.net/browse/CDAP-17470
https://cdap.atlassian.net/browse/CDAP-17382
https://cdap.atlassian.net/browse/CDAP-16814

Build: https://builds.cask.co/browse/CDAP-UDUT919

### CODE REVIEW

**Review by njbriggs — APPROVED** [review]

File: cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/index.tsx [hunk 1]
```diff
@@ -173,7 +176,7 @@ const MappingsView: React.FC<IMappingsProps> = ({
   const [loading, setLoading] = React.useState(true);
   const [rerunAssessment, setRerunAssessment] = React.useState(false);
 
-  function assessTable() {
+  function assessTable(updatedColumns?) {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/index.tsx:179]: TIL: you can use `?` to specify optional field without a type!. Nice, but looks like incomplete ternary operator or null coalescing 😄

File: cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/TableScatterPlotGraph/scatterPlot.ts [hunk 2]
```diff
@@ -24,6 +24,8 @@ export const COLOR_MAP = {
   activeOutline: '#9FC0FB',
   inactive: '#999999',
   inactiveOutline: '#686868',
+  error: '#D40001',
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/TableScatterPlotGraph/scatterPlot.ts:27]: We can import the colors from `colors.ts`?

**Review by ajainarayanan — APPROVED** [review]
Minor comment. Otherwise 👍 LGTM

File: cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/TableScatterPlotGraph/scatterPlot.ts (hunk 2 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/TableScatterPlotGraph/scatterPlot.ts:27]: These are custom colors for the graphs. Once we settled on these colors variance for graphs, I will move them into colors.ts and have graph specific colors there

File: cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/index.tsx (hunk 1 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/components/Replicator/Create/Content/Assessment/TablesAssessment/Mappings/index.tsx:179]: with the new null coalescing, the code make it seemed like the programmer is unsure about everything with question marks everywhere 😂

File: cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/Metrics/index.tsx [hunk 3]
```diff
@@ -102,31 +124,65 @@ const MetricsView: React.FC<WithStyles<typeof styles>> = ({ classes }) => {
...
+  const minutes = Math.floor(latency / ONE_MIN_SECONDS);
+  const seconds = latency % ONE_MIN_SECONDS;
+
+  return {
+    minutes,
+    seconds,
+  };
+}
+
+function formatNumberDisplay(num) {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Replicator/Detail/Monitoring/Metrics/index.tsx]: Seems like there is a similar function added to the replication utility. May be merge them?

File: cdap-ui/app/cdap/components/Replicator/Detail/TopPanel/ActionButtons/Configure/ConfigureModeless.tsx [hunk 4]
```diff
@@ -47,10 +49,21 @@ const styles = (theme): StyleRules => {
...
+    taskMemory: {
+      width: '200px',
+      margin: '15px 0 25px',
+    },
+    body: {
       padding: '10px 15px',
     },
+    content: {
+      height: '300px',
+      // padding: '10px 15px',
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/Replicator/Detail/TopPanel/ActionButtons/Configure/ConfigureModeless.tsx]: nit: remove?

**Review by ajainarayanan — APPROVED** [review]
Minor comment. 👍 LGTM

**Review by ajainarayanan — APPROVED** [review]

**Pull request merged**